metpy/plots/declarative.py

```diff
@@ -928,6 +933,78 @@ def draw(self):
             self._need_redraw = False
 
 
+@exporter.export
+class SurfaceStationPlot(HasTraits):
+    """Plot surface station data from a Pandas Dataframe"""
+
+    parent = Instance(Panel)
+    _need_redraw = Bool(default_value=True)
+
+
+    field = List(Unicode())
+    field.__doc__ = """The variables that will be plotted on the stations
+
+
+    This is a list of variables. Typical variables include temperature, dewpoint,
+    mean_sea_level_pressure, skyc (skycover)
+    """
+
+
+    dattim = Instance(datetime, allow_none=True)
+    field.__dattim__ = """Datetime object that will easily allow user to subset data
+
+    Make sure this is a datetime object, and by default, the time range will be set
+    to one hour
+    """
+
+
+    fontsize = Unicode()
+    field.__fontsize__ = """Desired fontsize for the fonts
```

Comment:
Should be `fontsize.__doc__` and would best to be a float or integer type. Also a default value?